template/docs/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Provide a narrative description of what you are trying to accomplish.
```

Comment:
Can we add something like this at the top:

 - [ ] I was not able to find an [open](https://github.com/zendframework/maintainers/issues?q=is%3Aopen) or [closed](https://github.com/zendframework/maintainers/issues?q=is%3Aclosed) issue matching what I'm seeing.
- [ ] This is not a question as they should be asked on [slack](https://zendframework.slack.com/) ([Signup for Slack here](https://zendframework-slack.herokuapp.com/)) or our [forums](https://discourse.zendframework.com/).

*swap `zendframework/maintainers` with `zendframework/<project>`*

Reply:
Done!